File: cineManagement/cineManagementDatabaseFirst/Models/DTOs/PeliculaConDetallesDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace cineManagementDatabaseFirst.Models.DTOs
{
    public class PeliculaConDetallesDTO
    {
        public int PeliculaId { get; set; }
        public string Nombre { get; set; }
        public string Descripcion { get; set; }
        public int? Duracion { get; set; }
        public string Imagen { get; set; }
        public string SalaNombre { get; set; }
        public int? EstadoSala { get; set; }
        public DateTime FechaPublicacion { get; set; }
        public DateTime FechaFin { get; set; }
    }
}

File: cineManagementFront/src/app/models/dtos/peliculaConDetalles.dto.ts
export interface PeliculaConDetallesDTO {
  peliculaId: number;
  nombre: string;
  descripcion: string;
  duracion?: number;
  imagen: string;
  salaNombre: string;
  estadoSala?: number;
  fechaPublicacion: Date;
  fechaFin: Date;
}
